person.rb

```diff
@@ -2,7 +2,7 @@
 
 # Person Class
 class Person < Nameable
```

Comment:
- [ ] Kindly add the method for adding rentals. You can take the approach suggested earlier: Import Rental class and note the order of the parameters

Reply:
This issue is well resolved! ✔️